### PR TITLE
Components

### PR DESCRIPTION
**Pull request by sntgopl**
# Request Review

## In this milestone I have:

- Created a new react app with `npx create-react-app`.
- Created a new Components directory with each component to be displayed and called by `App.js`.
- Imported all necessary dependencies to work with SPA and Router with React.
- Give basic style with CSS to `index.js`.

### CODE REVIEW

File: src/Components/Books.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const Book = () => (
```
Comment (mmsesay) [inline src/Components/Books.js]: - [ ] According to the requirements for this project, a single Book component should the book data as a prop and renders the title and the author.

Example: 
```js
const Book = ({ title, author}) => (

<div className="Book">
     <div>
           <h3>Action</h3>
           <h2>{title}</h2>
           <p>{author}</p>
     </div>
    // other code snippets
 </div>
)
```

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+);
+
+const Categories = () => (
+  <div>
+    <ul>
+      <h3>Categories</h3>
+      <li>All</li>
+    </ul>
+  </div>
+);
```
Comment (mmsesay) [inline src/App.js]: - [OPTIONAL] You can put these functions into a separate files as components.

File: src/App.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    <Link to="/categories">Categories</Link>
+  </nav>
+);
+
+const Books = () => (
+  <div className="HomePage">
+    <ul className="Booklist">
+      <Book />
+      <Book />
+    </ul>
```
Comment (mmsesay) [inline src/App.js]: - [ ] The book component should have a title and author props passed.
```js
<Book  title="book-title" author="book-author" />
```

- [OPTIONAL] You can make a small array of objects which hold a few books loop through them and render the book component with the book props passed.

Example:
```js
const tempBooks = [
         {id: 1, title: "Book 1", author: "author 1"},
         {id: 2, title: "Book 2", author: "author 2"}
];

<ul className="Booklist">
{tempBooks.map((book) => (
   <div key={book.id}>
       <Book  title={book.title} author={book.author} />
    </div>
)}
</ul>

```

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @sntgopl, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
 - Good Readme :heavy_check_mark: 
 - No linter error :heavy_check_mark: 
 - Good PR title :heavy_check_mark: 
 - Good PR summary :heavy_check_mark:  


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 2 above)
Comment (sntgopl) [reply src/App.js]: I will put each page in a different file. Thank you.

File: src/App.js (hunk 3 above)
Comment (sntgopl) [reply src/App.js]: I created an array of objects to map and display each book.

File: src/Components/Books.js (hunk 1 above)
Comment (sntgopl) [reply src/Components/Books.js]: Thank you for the advice. I changed it and pass it as props.

**Review by jaflih — APPROVED** [review]
Hi @sntgopl   , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

Good Readme ✔️
No linter error ✔️
Good PR title ✔️
Good PR summary ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @jaflih in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**